src/server/object_services/schemas/object_md_indexes.js

```diff
@@ -115,6 +115,7 @@ module.exports = [
         // aggregate_objects_by_delete_dates()
         fields: {
             deleted: 1,
+            reclaimed: 1,
```

Comment:
Why did you add reclaimed to the indexed fields?
Is it needed for md-aggregator?

Reply:
Maybe you meant to add `find_unreclaimed_objects()` to the list of functions using this index?